src/sql/workbench/services/notebook/sql/sqlSessionManager.ts

```diff
@@ -25,7 +25,7 @@ import * as notebookUtils from 'sql/parts/notebook/notebookUtils';
 import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
 
 export const sqlKernel: string = localize('sqlKernel', 'SQL');
-export const sqlKernelError: string = localize("sqlKernelError", "SQL kernel error");
+export const sqlKernelError: string = localize('sqlKernelError', 'SQL kernel error');
```

Comment:
You should use double quotes when localizing strings `localize('sqlKernelError', "SQL kernel error")`

Reply:
Why, out of curiosity?

Reply:
Could we get a linting rule on this one? I've messed this up a lot, and looking at the code this is wildly inconsistent.

Reply:
Its just a way to distinguish between what strings are visible on the UI (should be localized) and which ones aren't.

Reply:
Roger that